packages/graphback-core/src/index.ts

```diff
@@ -8,12 +8,15 @@ export * from './plugin/GraphbackGlobalConfig'
 export * from './plugin/GraphbackCRUDGeneratorConfig';
 export * from './plugin/ModelDefinition'
 export * from './plugin/getSelectedFieldsFromResolverInfo';
-export * from './utils/printSchemaWithDirectives';
-export * from './utils/metadataAnnotations';
 export * from './plugin/GraphbackCoreMetadata'
+
 export * from './relationships/RelationshipMetadataBuilder';
 export * from './relationships/relationshipHelpers';
-export * from './runtime'
-export * from './db';
 export * from './annotations/DefaultValueAnnotation';
 
+export * from './utils/printSchemaWithDirectives';
+export * from './utils/metadataAnnotations';
+export * from './utils/fieldTransformHelpers';
+
+export * from './runtime'
```

Comment:
```suggestion
export * from './runtime';
```

Reply:
Noticed that sometime we add a semicolon, sometimes we don't. This an be applied or ignored as you see fit but I think we should be consistent.

Reply:
when is vscode it adds it but devs do not add it :)
I really hate eslint package. It is so convoluted and it doesn't format code right. 
Using prettier will be much better but using both at the same time gets tricky as styles need to align. I will create issue to see if we can reformat all code using prettier and then make it pass using lint

Reply:
Cool, thanks.